catalog/opensearch/catalog-opensearch-source/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -56,8 +56,13 @@
             <property name="shortname" value="DDF-OS"/>
             <property name="endpointUrl"
                       value="https://localhost:8993/services/query"/>
+            <property name="authenticationType" value="saml"/>
             <property name="username" value=""/>
             <property name="password" value=""/>
+            <property name="oauthDiscoveryUrl" value="https://localhost:8443/auth/realms/master/.well-known/openid-configuration"/>
```

Comment:
It is the same one but I don't get what you mean by one central location. These are blueprint defaults.

Reply:
By central I meant "in one place", so if you wanted to go in and change the default value you would only have to do it once, rather than jump around a couple of files.

But its fine the way it is; I wouldn't spend too much time improving on this.